publiccode.yml

```diff
@@ -0,0 +1,89 @@
+# This repository adheres to the publiccode.yml standard by including this
+# metadata file that makes public software easily discoverable.
+# More info at https://github.com/italia/publiccode.yml
+
+publiccodeYmlVersion: '0.2'
+name: 'itowns'
+url: 'https://github.com/iTowns/itowns'
+landingURL: 'https://github.com/iTowns/itowns.git'
+softwareVersion: "2.44.2" # needs to be updated
+releaseDate: '2024-09-17'# needs to be updated
+softwareType: library
+logo: "https://github.com/iTowns/itowns/blob/master/examples/images/itowns_logo.png"
+inputTypes: # ToDo : complete
+  - application/json
+platforms:
+  - web
+categories:
+  - website-builder
+  - data-visualization
+  - it-development
+usedBy: # ToDo : complete and ask for permissions
+  - IGN (https://www.ign.fr/)
+  - Ciril Grp (https://www.ciril.net/fr/)
+  - Sogelink (https://www.sogelink.com/)
+  - Laboratoire Liris (https://liris.cnrs.fr/)
+  - Laboratoire Lastig (https://www.umr-lastig.fr/)
+roadmap: "https://www.itowns-project.org/"
+developmentStatus: stable
+intendedAudience:
+  scope:
```

Comment:
Reading the list of [scopes](https://docs.italia.it/italia/developers-italia/publiccodeyml-en/en/master/scope-list.html#scope-list), I'm hoping that we could update it at the end of the year with `culture`. :P

@LoicUTC Any thoughts?